Automattic/jetpack
# Protect: Introduce Jetpack Protect plugin initial skeleton

This PR may overlap with #23558. I tested that one and it threw fatals. Turned off Jetpack as I assumed that was the issue, but then didn't check much of the structure. Will check how different the generated structure is after #23434 got merged

### Disconnected view
<img width="1788" alt="image" src="https://user-images.githubusercontent.com/746152/159915292-3d3262ac-92c1-475f-8424-cf39e949ada6.png">

### Connected view

<img width="1787" alt="image" src="https://user-images.githubusercontent.com/746152/159915588-ef209b50-69e4-4908-a90d-15ca64526340.png">



#### Changes proposed in this Pull Request:

* Adds plugin initial structure based on the new Jetpack generator introduced in #23434

#### Jetpack product discussion


#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:

* Launch a [JN site with this branch and plugin](https://jurassic.ninja/create/?jetpack-beta&nojetpack&branches.jetpack-protect=add/security-plugn-named-jetpack-protect)
* *[](https://user-images.githubusercontent.com/746152/157232961-d07a5a23-4d00-4dcb-aae1-356adba2a132.png) (This link launches without the main Jetpack plugin).
* Visit the Jetpack Protect menu
* Connect the site. 
* Expect to se the My Jetpack menu item
